src/components/FindWallet/WalletTable.tsx

```diff
@@ -0,0 +1,1099 @@
+    category: "security",
+    icon: <NonCustodial />,
+  },
+  {
+    label: "Hardware wallet support",
+    value: "Hardware wallet support",
+    filterKey: "hardware_support",
+    category: "feature",
+    icon: <HardwareSupport />,
+  },
+  {
+    label: "WalletConnect",
+    value: "WalletConnect",
+    filterKey: "walletconnect",
+    category: "feature",
+    icon: <WalletConnect />,
+  },
+  {
+    label: "RPC importing",
+    value: "RPC importing",
+    filterKey: "rpc_importing",
+    category: "feature",
+    icon: <RPCImporting />,
+  },
+  {
+    label: "NFT support",
+    value: "NFT support",
+    filterKey: "nft_support",
+    category: "feature",
+    icon: <NFTSupport />,
+  },
+  {
+    label: "Connect to dapps",
+    value: "Connect to dapps",
+    filterKey: "connect_to_dapps",
+    category: "feature",
+    icon: <ConnectDapps />,
+  },
+  {
+    label: "Staking",
+    value: "Staking",
+    filterKey: "staking",
+    category: "feature",
+    icon: <Staking />,
+  },
+  {
+    label: "Swaps",
+    value: "Swaps",
+    filterKey: "swaps",
+    category: "feature",
+    icon: <Swap />,
+  },
+  {
+    label: "Layer 2",
+    value: "Layer 2",
+    filterKey: "layer_2",
+    category: "feature",
+    icon: <Layer2 />,
+  },
+  {
+    label: "Gas fee customization",
+    value: "Gas fee customization",
+    filterKey: "gas_fee_customization",
+    category: "feature",
+    icon: <GasFeeCustomization />,
+  },
+  {
+    label: "ENS support",
+    value: "ENS support",
+    filterKey: "ens_support",
+    category: "feature",
+    icon: <ENSSupport />,
+  },
+  {
+    label: "Token importing",
+    value: "Token importing",
+    filterKey: "erc_20_support",
+    category: "feature",
+    icon: <ERC20Support />,
+  },
+  {
+    label: "EIP-1559 support",
+    value: "EIP-1559 support",
+    filterKey: "eip_1559_support",
+    category: "feature",
+    icon: <Eip1559 />,
+  },
+  {
+    label: "Buy crypto",
+    value: "Buy crypto",
+    filterKey: "buy_crypto",
+    category: "trade_and_buy",
+    icon: <BuyCrypto />,
+  },
+  {
+    label: "Withdraw crypto",
+    value: "Withdraw crypto",
+    filterKey: "withdraw_crypto",
+    category: "trade_and_buy",
+    icon: <WithdrawCrypto />,
+  },
+  {
+    label: "Multisig",
+    value: "Multisig",
+    filterKey: "multisig",
+    category: "smart_contract",
+    icon: <Multisig />,
+  },
+  {
+    label: "Social recovery",
+    value: "Social recovery",
+    filterKey: "social_recovery",
+    category: "smart_contract",
+    icon: <SocialRecover />,
+  },
+]
+
+const WalletTable = ({ data, filters, walletData }) => {
+  const [walletCardData, setWalletData] = useState(
+    walletData.map((wallet) => {
+      return { ...wallet, moreInfo: false, key: wallet.image_name }
+    })
+  )
+  const [firstFeatureSelect, setFirstFeatureSelect] = useState(
+    featureDropdownItems[1]
+  )
+  const [secondFeatureSelect, setSecondFeatureSelect] = useState(
+    featureDropdownItems[13]
```

Comment:
```suggestion
    featureDropdownItems[14]
```
This should be "Buy cryto" not "EIP-1559 support".